pr_testing/test_multiple_prs.sh

```diff
@@ -1462,6 +1477,7 @@ if [ "X$DO_SHORT_MATRIX" = Xtrue ]; then
   fi
   if $PRODUCTION_RELEASE ; then
     for ex_type in ${EXTRA_RELVALS_TESTS} ; do
+      [ $ex_type = "GPU" ] && (mark_commit_status_all_prs "gpu" "success" -u "${BUILD_URL}" -d "Legacy status" -e; continue)
```

Comment:
@iarspider , this is not needed. All previous statuses are reset at https://github.com/cms-sw/cms-bot/blob/master/pr_testing/test_multiple_prs.sh#L219

Reply:
It is set to "pending" there, but we no longer update "gpu" status (only "cuda", "rocm", ...). I think removing this line will make the bot wait until it completes - which will never happen.

Reply:
I'm checking if my understanding of bot logic is correct